airbyte-webapp/packages/stylelint-plugin/no-color-variables-in-rgba.js

```diff
@@ -0,0 +1,35 @@
+// eslint-disable-next-line @typescript-eslint/no-var-requires
+const stylelint = require("stylelint");
+const { ruleMessages } = stylelint.utils;
+const ruleName = "airbyte/no-color-variables-in-rgba";
+const messages = ruleMessages(ruleName, {
+  variableFoundInRgba: () => `A color variable should not be used within an rgba() function.`,
```

Comment:
Maybe we should be more explicit here that we "can't" use them inside instead of just "shouldn't", since they basically won't work. I also wonder if we can reference to any place that explains why this is the case?

Reply:
Good point, updated to change to "can't".

As for an explanation, I thought about linking either to this PR in the error message (I could add some more context in the description) or to the `STYLEGUIDE.md`. Any preference? I would add a new section in the style guide, `## Styling`

Reply:
I think as well the Styleguide might be the best place for now to put this and link to.

Reply:
Cool, added!